subprojects/core/src/integTest/groovy/org/gradle/api/tasks/AbstractDirectorySensitivityIntegrationSpec.groovy

```diff
@@ -483,7 +479,7 @@ abstract class AbstractDirectorySensitivityIntegrationSpec extends AbstractInteg
                 @Override
                 void transform(TransformOutputs outputs) {
                     File zippedFile = getZippedFile().get().getAsFile()
-                    File unzipDir = outputs.dir(zippedFile.getName())
+                    File unzipDir = outputs.dir(zippedFile.getName() + "/unzipped")
```

Comment:
Why is this now in the `/unzipped` subdirectory? Is this important for the test?

Reply:
It's not important, it just makes it easier to identify when debugging/troubleshooting since it's more clear that this is the unzipped output directory.  That's all.

Reply:
👍